Sources/App/CRP/GitHubService+CRP.swift

```diff
@@ -0,0 +1,38 @@
+import Vapor
+import Stevenson
+
+extension GitHubService {
```

Comment:
Not sure if this is really only tighlty coupled with CRP only. I mean if in the future we want to use this `makeGitHubRelease` to create a GitHub Release on github.com (containing the Changelog and all) we could also use that as well. If I'm not mistaken, there's nothing in this code using anything specific to CRP, only things specific to how to interpret our branch names as being app names and version numbers, right?

`GitHubService+Release.swift` instead maybe?

Reply:
I recon we can rename it when scope when it is needed actually changes

Reply:
Sure but even now there is nothing related to CRP in that code ;)

Reply:
I rammed the file to `GithubService+Release`